src/pages/dashboard.tsx

```diff
@@ -13,4 +15,19 @@ const CustomerDashboard: NextPage = () => {
   );
 };
 
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const session = await getServerAuthSession(context);
+
+  if (session && !(session.user.type == UserType.CUSTOMER)) {
```

Comment:
Shouldn't all the equalities be `===`?

```suggestion
  if (session && !(session.user.type === UserType.CUSTOMER)) {
```

Reply:
Updated